collections/maps.go

```diff
@@ -26,4 +29,17 @@ func Keys(m map[string]string) []string {
 	sort.Strings(out)
 
 	return out
-}
\ No newline at end of file
+}
+
+// Flatten returns a string slice with key=value items, sorted alphabetically
+func Flatten(m map[string]string) []string {
```

Comment:
I'm not sure `Flatten` is the right name for this function. Maybe this should be called `KeyValueStringSlice` or `CreateKeyEqualValueSlice`?

I think flat maps are usually a map that only has one level, and `flatten` is the function that removes the nested levels. E.g., 

- https://stackoverflow.com/questions/54183766/how-to-flatten-a-hashmap
- https://www.freecodecamp.org/news/how-to-flatten-a-dictionary-in-python-in-4-different-ways/

Reply:
Fixed in e7c3a9194b17b07874c957021f6f31cf39d27594